src/Microsoft.ML.Console/Microsoft.ML.Console.csproj

```diff
@@ -28,13 +28,14 @@
     <ProjectReference Include="..\Microsoft.ML.TensorFlow\Microsoft.ML.TensorFlow.csproj" />
     <ProjectReference Include="..\Microsoft.ML.Transforms\Microsoft.ML.Transforms.csproj" />
 
-    <NativeAssemblyReference Include="FastTreeNative" />
-    <NativeAssemblyReference Include="CpuMathNative" />
+    <NativeAssemblyReference Condition="'$(TargetArchitecture)' != 'arm64' And '$(TargetArchitecture)' != 'arm'" Include="FastTreeNative" />
+    <NativeAssemblyReference Condition="'$(TargetArchitecture)' != 'arm64' And '$(TargetArchitecture)' != 'arm'" Include="CpuMathNative" />
     <NativeAssemblyReference Include="MatrixFactorizationNative" />
     <NativeAssemblyReference Include="LdaNative" />
-    <NativeAssemblyReference Include="SymSgdNative"/>
-    <NativeAssemblyReference Include="MklImports"/>
-    <NativeAssemblyReference Condition="'$(OS)' == 'Windows_NT'" Include="libiomp5md"/>
+    <!-- TODO: once we fix the 4 intel MKL methods, SymSgdNative will need to go back in. -->
```

Comment:
How are we tracking this TODO?

Reply:
I have it tracked locally, but I will make an issue for it. There are 4 methods that deal with vector products that just need to be replaced so SymSgd can work on arm.